From 18f7ccc9251ab0fdf14f031aef596d016aab282b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mateusz=20Miku=C5=82a?= <mati865@gmail.com>
Date: Sun, 5 Nov 2023 22:41:59 +0100
Subject: [PATCH] Build mingw-w64 for Windows 7

---
 scripts/build/libc/mingw-w64.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/build/libc/mingw-w64.sh b/scripts/build/libc/mingw-w64.sh
index d4f3634f..9482004d 100644
--- a/scripts/build/libc/mingw-w64.sh
+++ b/scripts/build/libc/mingw-w64.sh
@@ -44,6 +44,7 @@ mingw_w64_headers() {
         --build=${CT_BUILD} \
         --host=${CT_TARGET} \
         --prefix=${MINGW_INSTALL_PREFIX} \
+        --with-default-win32-winnt=0x601 \
         "${sdk_opts[@]}"
 
     CT_DoLog EXTRA "Compile Headers"
-- 
2.34.1

